### PR TITLE
Manager kustomize

### PR DESCRIPTION
**Pull request by hunchback**
this is to solve the issue with kustomization.yaml content being changed by each operation

### CODE REVIEW

File: manager/Makefile [hunk 1]
```diff
@@ -56,12 +59,14 @@ charts-crd: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl manifests
 	$(TOOLBIN)/kustomize build config/crd > $(ROOT_DIR)/charts/m4d-crd/templates/crds.yaml
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+deploy: config/default/kustomization.yaml
```
Comment (roee88) [inline manager/Makefile:62]: why a separate line?

File: manager/Makefile [hunk 2]
```diff
@@ -56,12 +59,14 @@ charts-crd: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl manifests
...
 deploy: docker-secret manifests $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl
 	$(TOOLBIN)/kubectl create namespace m4d-system || true
 	cd config/manager && $(ABSTOOLBIN)/kustomize edit set image controller=${IMG}
 	$(TOOLBIN)/kustomize build config/default | $(TOOLBIN)/kubectl apply -f -
 
 # Delete controller in the configured Kubernetes cluster in ~/.kube/config
+undeploy: config/default/kustomization.yaml
```
Comment (roee88) [inline manager/Makefile:69]: same

Comment (hunchback) [reply manager/Makefile]: fixed

File: manager/Makefile (hunk 1 above)
Comment (hunchback) [reply manager/Makefile]: fixed

**Review by roee88 — APPROVED** [review]

**Pull request merged**